sklearn/linear_model/bayes.py

```diff
@@ -379,7 +379,7 @@ def _log_marginal_likelihood(self, n_samples, n_features, eigen_vals,
 # ARD (Automatic Relevance Determination) regression
 
 
-class ARDRegression(LinearModel, RegressorMixin):
+class ARDRegression(RegressorMixin, LinearModel):
```

Comment:
Some how from the names, I'd think the other way is the correct order.

Reply:
Why? Mixins go to the very left so the super call there doesn't go to ``object``. Clearly as Gaël argues here: https://github.com/scikit-learn/scikit-learn/issues/14044#issuecomment-520598893 that's much easier to reason about than having an actual class hierarchy....